lib/bot.js

```diff
@@ -362,6 +387,11 @@ class Bot {
       ircChannel: channel
     };
 
+    // Do not send to Discord if this user is on the ignore list.
+    if (this.ignoredIrcUser(author)) {
```

Comment:
maybe move this check to https://github.com/anirbanmu/discord-irc/blob/be281c5addff12a250866b351a05b91cb9d168e6/lib/bot.js#L377?

either in the same conditional or the line below - up to you

Reply:
Moved.